addon/services/page-title-list.js

```diff
@@ -89,7 +89,7 @@ export default Service.extend({
       return;
     }
 
-    var previous = this.tokens.slice(-1)[0];
+    let previous = this.tokens.slice(-1)[0];
```

Comment:
I found this snippet confusing at first. We could do something like`this.tokens.toArray().reverse()[0]` for clarity.